File: ApiCentroSalud/DTOs/TurnoRequest.cs
namespace ApiCentroSalud.DTOs
{
    public class TurnoRequest
    {
        public int PacienteId { get; set; }
        public int MedicoId { get; set; }
        public DateTime Fecha { get; set; }
        
        public string DetalleMotivo { get; set; }   
        public int MotivoId { get; set; }
    
    }

}

File: app/src/main/java/com/novacenter/app/data/model/TurnoRequest.kt
package com.novacenter.app.data.model

data class TurnoRequest(
    val pacienteId: Int,
    val medicoId: Int,
    val fecha: String, // formato ISO (ej: "2025-07-15T10:00:00")
    val detalleMotivo: String?,
    val motivoId: Int
)
